Mail/Template.php

```diff
@@ -205,10 +214,18 @@ public function loadDefault($templateId)
     public function getProcessedTemplate(array $variables = [])
     {
         $processor = $this->getTemplateFilter()
-            ->setUseSessionInUrl(false)
             ->setPlainTemplateMode($this->isPlain())
             ->setIsChildTemplate($this->isChildTemplate())
             ->setTemplateProcessor([$this, 'getTemplateContent']);
+        /**
+         * After Magento v2.3.5 session_id is removed from URLs
+         *
+         * @customization START
+         */
+        if (version_compare($this->sailthruSettings->getMagentoVersion(), '2.3.5', '<')) {
+            $processor->setUseSessionInUrl(false);
```

Comment:
Is this backwards? The comment makes me think it should be this
```suggestion
        if (version_compare($this->sailthruSettings->getMagentoVersion(), '2.3.5', '>')) {
            $processor->setUseSessionInUrl(false);
```

Reply:
this must not be backwards. setUseSessionInUrl() is deprecated after 2.3.5, thus we keep calling it only prior to v2.3.5